CONTRIBUTING.md

```diff
@@ -0,0 +1,95 @@
+
+Working on your first Pull Request? You can learn how from this *free* series, [How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github).
+
+At this point, you're ready to make your changes! Feel free to ask for help; everyone is a beginner at first :smile_cat:
+
+# Getting started
+
+1. Create your own fork of the code
+2. Do the changes in your fork
+3. If you like the change and think the project could use it:
+    * Be sure you have followed the code style for the project.
+    * Make sure `bundle exec rake ci` passes.
+
+As a rule of thumb, changes are obvious fixes if they do not introduce any new functionality or creative thinking. As long as the change does not affect functionality, some likely examples include the following:
+
+* Spelling / grammar fixes
+* Typo correction, white space and formatting changes
+* Comment clean up
+* Bug fixes that change default return values or error codes stored in constants
+* Adding logging messages or debugging output
+* Changes to ‘metadata’ files like Gemfile, .gitignore, build scripts, etc.
+* Moving source files from one directory or package to another
+
+# How to report a bug
+
+If you find a security vulnerability, do NOT open an issue. Email [pepibumur@gmail.com](mailto://pepibumur@gmail.com) instead.
+
+If you don’t want to use your personal contact information, set up a “security@” email address. Larger projects might have more formal processes for disclosing security, including encrypted communication.
+
+### How to file a bug report
+
+When filing an issue, make sure to answer these five questions:
+
+1. What version of Xcode/Swift are you using?
+2. What operating system and processor architecture are you using?
+3. What did you do?
+4. What did you expect to see?
+5. What did you see instead?
+
+# How to suggest a feature or enhancement
+
+Before suggesting a new feature, make sure that it aligns with the project philosophy, providing an API to interact with Xcode projects files.
+
+If you find yourself wishing for a feature that doesn't exist in xcodeproj, you are probably not alone. There are bound to be others out there with similar needs. Many of the features that xcodeproj has today have been added because our users saw the need. Open an issue on our issues list on GitHub which describes the feature you would like to see, why you need it, and how it should work.
+
+# Code review process
+
+The core team looks at Pull Requests on a regular basis.
+
+After feedback has been given we expect responses within two weeks. After two weeks we may close the pull request if it isn't showing any activity.
+
+# Community
+
+You can chat with the core team on Slack. You can join the group on the [following link](https://swift-xcode.herokuapp.com).
+
+# Conventions
+
+### Commit message
+- Commit message should include the issue number together with the commit message: `[#number] commit message`.
```

Comment:
What if I commit 10 times? Every message should be in this format, right?

Reply:
Yes, it makes it easier to find the issue the commit belonged to, and get some context on that issue.